docs/index.rst

```diff
@@ -1,51 +1,54 @@
 Overview
 --------
 
-Streamlink is a :ref:`command-line utility <cli>` that pipes video streams
+Streamlink is a :ref:`command-line utility <cli>` which pipes video streams
 from various services into a video player, such as `VLC`_.
-The main purpose of Streamlink is to allow the user to avoid buggy and CPU
-heavy flash plugins but still be able to enjoy various streamed content.
+The main purpose of Streamlink is to avoid resource-heavy and unoptimized websites,
+while still allowing the user to enjoy various streamed content.
 There is also an :ref:`API <api_guide>` available for developers who want access
-to the video stream data. This project was forked from Livestreamer, which is
-no longer maintained.
+to the stream data.
 
-- Latest release: |version| (https://github.com/streamlink/streamlink/releases/latest)
-- GitHub: https://github.com/streamlink/streamlink
-- Issue tracker: https://github.com/streamlink/streamlink/issues
-- PyPI: https://pypi.org/project/streamlink/
-- Free software: Simplified BSD license
+This project was forked from Livestreamer, which is no longer maintained.
```

Comment:
this line could probably be removed, seems at this point unimportant

Reply:
It may not have any practical reasons, but I think it's important to briefly mention the origins of the project and should therefore stay on the index page.